setup.py

```diff
@@ -22,7 +22,9 @@
     install_requires=[
         "contextdecorator>=0.10.0",
         "inflection>=0.3.1",
+        "jaeger-client>=4.0.0",
```

Comment:
At the risk of adding extract work, this sounds like an optional requirement. If `microcosm` is meant to be open-source (tm), this kind of dependency feels like it ought to be optional.

Reply:
Good call. I'll make it optional.

Reply:
So I tried making it optional and without a good example to go off of, I want to make it required for now, and downgrade it to optional when we get such a request. Making it optional isn't hard, but deciding on the conventions around that is something I don't want to do now.